dprint/app.json

```diff
@@ -1,6 +1,14 @@
 {
 	"lineWidth": 120,
 	"useTabs": true,
+	"prettier": {
+		"associations": [
+			"**/*.{css,scss}"
+		],
```

Comment:
Do we need to ignore `node_modules`?

Reply:
It's already ignored, see `excludes` below